app/src/main/java/turtler/voyageur/fragments/DetailEvent.java

```diff
@@ -0,0 +1,330 @@
+    Unbinder unbinder;
+    Location mLastLocation;
+    Image image;
+    Event currentEvent;
+    Dialog d;
+    Bitmap selectedImageBitmap;
+    GoogleApiClient mGoogleApiClient;
+    private TransferUtility transferUtility;
+    public final static int CAPTURE_IMAGE_ACTIVITY_REQUEST_CODE = 1034;
+    public final static int PICK_PHOTO_CODE = 1046;
+    public String photoFileName = "photo";
+
+    private static final String ARG_EVENT_ID = "eventId";
+
+    // TODO: Rename and change types of parameters
+    private String eventId;
+
+
+    public DetailEvent() {
+    }
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment using the provided parameters.
+     *
+     * @param eventId Parameter 1.
+     * @return A new instance of fragment DetailEvent.
+     */
+    // TODO: Rename and change types and number of parameters
+    public static DetailEvent newInstance(String eventId) {
+        DetailEvent fragment = new DetailEvent();
+        Bundle args = new Bundle();
+        args.putString(ARG_EVENT_ID, eventId);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (getArguments() != null) {
+            eventId = getArguments().getString(ARG_EVENT_ID);
+            ParseQuery<Event> query = ParseQuery.getQuery("Event");
+            query.getInBackground(eventId, new GetCallback<Event>() {
+                @Override
+                public void done(Event eventObj, ParseException e) {
+                    if (e == null) {
+                        currentEvent = eventObj;
+                        d.setTitle(currentEvent.getTitle());
+                        setEventDetailLayout(currentEvent);
+                    }
+                }
+            });
+        }
+
+        transferUtility = AmazonUtils.getTransferUtility(getContext());
+    }
+
+    public void setEventDetailLayout(final Event ev) {
+        ArrayList<Image> images = new ArrayList<>();
+        try {
+            images = (ArrayList<Image>) ev.imagesRelation().getQuery().find();
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        if (images.size() > 0) {
+            for (Image image : images) {
+                TextSliderView textSliderView = new TextSliderView(getContext());
+                textSliderView.image(image.getPictureUrl());
+                sliderLayout.addSlider(textSliderView);
+
+            }
+            if (images.size() == 1) {
+                sliderLayout.stopAutoCycle();
+            }
+        }
+        sliderLayout.setVisibility(View.VISIBLE);
+
+        tvCaption.setText(ev.getCaption());
+        if (ev.getDate() != null) {
+            tvDate.setText(TimeFormatUtils.dateTimeToString(ev.getDate()));
+        }
+        /*
+        tvTitle.setOnClickListener(new View.OnClickListener() {
```

Comment:
we still need some way to connect to the map marker though...was planning on adding that back in some form in a future PR